src/components/popover/popover.ts

```diff
@@ -94,10 +94,6 @@ export class Popover extends Component<PopoverProps, SpreadsheetChildEnv> {
       this.currentDisplayValue = newDisplay;
 
       if (!anchor) return;
-      el.style.top = "";
```

Comment:
useless diff ?

Reply:
I don't think so. It's commit https://github.com/odoo/o-spreadsheet/commit/c49ed077803c694198655bbe9d67264bf3780d81 that's being reverted

Reply:
yup